### PR TITLE
More events

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

File: lib/protocol.schema.json [hunk 1]
```diff
@@ -15,6 +15,14 @@
                     },
                     "required": [ "response" ]
                 },
+                "Sent": {
```
Comment (kpp) [inline lib/protocol.schema.json]: MessageSent?

Comment (suhr) [reply lib/protocol.schema.json]: Would be fine too.

Comment (kpp) [reply lib/protocol.schema.json]: Rename it please

Comment (suhr) [reply lib/protocol.schema.json]: Done.

**Pull request merged**